pkgs/applications/networking/instant-messengers/wavebox/default.nix

```diff
@@ -1,16 +1,17 @@
-{ stdenv, fetchurl, makeDesktopItem, makeWrapper, autoPatchelfHook
-, xorg, gtk2, gtk3 , gnome2, gnome3, nss, alsaLib, udev, libnotify, xdg_utils }:
+{ alsaLib, autoPatchelfHook, fetchurl, gtk3, libnotify,
+  makeDesktopItem, makeWrapper, nss, stdenv, udev, xdg_utils,
+  xorg }:
```

Comment:
If you're changing this, could you please update it to use the [style rules](https://nixos.org/nixpkgs/manual/#sec-syntax) in the manual? Thanks!

Reply:
Apologies, @alyssais. Hopefully that's inline with the manual now :+1:

Reply:
No need to apologise! Thanks for the fix